packages/core/admin/admin/src/StrapiApp.tsx

```diff
@@ -283,6 +285,14 @@ class StrapiApp {
     });
   };
 
+  addRBACMiddleware = (m: RBACMiddleware | RBACMiddleware[]) => {
+    if (Array.isArray(m)) {
+      this.rbac.use(m);
+    } else {
+      this.rbac.use(m);
```

Comment:
sorry, I can't find the difference between these two flows, they both call
`this.rbac.use(m);`

Reply:
No, TS was being weird.